Foundation/NSString.swift

```diff
@@ -20,6 +20,11 @@ extension unichar : ExpressibleByUnicodeScalarLiteral {
     }
 }
 
+// Placeholder for a future implementation
+public func NSLocalizedString(_ key: String, _ comment: String?) -> String {
```

Comment:
This is, I believe, not the correct signature for NSLocalizedString in Swift. For one, “comment” should be an argument label (i.e., `comment:` and not `_ comment:`), but there are also other parameters with default values that are not reflected here. I understand it is a placeholder, but the signature should match Darwin Foundation.

Reply:
Also, if we're going to add it we may as well put in the placeholder as it's done in Darwin:

```
/// Returns a localized string, using the main bundle if one is not specified.
public func NSLocalizedString(_ key: String, tableName: String? = default, bundle: Bundle = default, value: String = default, comment: String) -> String
```

Reply:
Done. Seems to work